src/components/com_tc/administrator/models/content.php

```diff
@@ -480,7 +480,7 @@ public function getGlobalTCIdList()
 		$query->from($db->quoteName('#__tc_patterns', 'p'));
 		$query->join('LEFT', $db->quoteName('#__tc_content', 'c') . ' ON (' . $db->quoteName('c.tc_id') . ' = ' . $db->quoteName('p.tc_id') . ')');
 		$query->where($db->quoteName('c.global') . " = " . $db->quote(1));
-		$query->where($db->quoteName('c.start_date') . " <= " . $db->quote($today));
+		$query->where($db->quoteName('c.start_date') . " < " . $db->quote($today));
```

Comment:
@paritshivani Why you have removed equal to, we should display t&c if date is equal to current date time